### ISSUE TITLE
Moving data file upload limit to backend

### ISSUE THREAD

**Issue by dorisjlee** [zenvisage/zenvisage#190]
For security issues, we should move data file upload limit to backend, as suggested by @siddta . Inside fileUploader.js.
```
if($cookies.getObject('userinfo') || !login_ava){
      formData = new FormData(this);
      if (formData.get("csv").name.split(".").pop() != "csv" ){
        alert("Please select a csv file");
        return;
      }
      else if (formData.get("csv").size > 100000000) {
        alert("Do not upload files over 100MB");
        return;
      }
      parseCSV(formData);
      datasetNameInput = $("#datasetNameInput").val();
      console.log('test:',$(this).attr('action'),$(this).attr('method'));
      log.info("dataset upload: ",$("#datasetNameInput").val())
      // $('#uploaderModal').modal('toggle');
      document.getElementById("uploadingProgressMessage").style.display = "block";
      document.getElementById("submitButton").style.display = "none";
    }else{
      alert("Please log in first to upload dataset.")
    }
```

**Comment by dorisjlee** [comment]
assigned to Xu

**Comment by dorisjlee** [comment]
Resolved by @xuy2 on commit [3a7c90e](https://github.com/zenvisage/zenvisage/commit/3a7c90e627bb7d367770525a3bfdfb2590510998)
